crates/miden-testing/tests/integration/scripts/in_flight_swap.rs

```diff
@@ -0,0 +1,239 @@
+use miden_crypto::{Word, rand::FeltRng};
+use miden_lib::{
+    account::wallets::{AuxWallet, BasicWallet},
+    note::utils::{build_p2id_recipient, build_swap_tag},
+    transaction::TransactionKernel,
+};
+use miden_objects::{
+    Felt,
+    account::{Account, AccountId},
+    asset::Asset,
+    crypto::rand::RpoRandomCoin,
+    note::{
+        Note, NoteAssets, NoteDetails, NoteExecutionHint, NoteInputs, NoteMetadata, NoteRecipient,
+        NoteScript, NoteTag, NoteType,
+    },
+    transaction::OutputNote,
+};
+use miden_testing::{AccountState, Auth, MockChain};
+use rand::random;
+
+#[test]
+fn test_inflight_swap() {
```

Comment:
Nit: to match the other name of the test
```suggestion
fn test_in_flight_swap() {
```